test/_helpers/ignoreSVGStrings.jsx

```diff
@@ -1,6 +0,0 @@
-// Don't load svg strings into tests
-require.extensions['.svg'] = (obj) => {
```

Comment:
`require.extensions` is node.js specific (does not work in webpack/browser). Though, it also seems it's not necessary anymore as enzyme fixed support for SVGs (https://github.com/airbnb/enzyme/issues/375)?

Reply:
The enzyme bug was on svg elements being present; this file makes svg files requirable in node, which is a different thing.

Reply:
Thanks for clarifying. Then isn't what this is doing as well https://github.com/airbnb/react-dates/blob/master/.babelrc#L6? the `.svg` imports won't get to node.js - they'll be transformed before that in babel.